ST_species.c

```diff
@@ -429,21 +310,14 @@ void Species_Annual_Kill(const SppIndex sp, int killType)
 				? ((x)>-xF_DELTA && (x)<xF_DELTA) \
 						: ((x)>-xD_DELTA && (x)<xD_DELTA) )
 
```

Comment:
@chaukap again with `ZERO` removed, we can remove the macro within this function:

```
#define xF_DELTA (20*F_DELTA)
#define xD_DELTA (20*D_DELTA)
#define ZERO(x) \
		( (sizeof(x) == sizeof(float)) \
				? ((x)>-xF_DELTA && (x)<xF_DELTA) \
						: ((x)>-xD_DELTA && (x)<xD_DELTA) )
```

Reply:
@kpalmqui Good catch I took them out.

Reply:
@chaukap great - everything looks good and ready to merge!